### DISCUSSION TITLE
Give priority to answering questions from users who have starred in this project

### DISCUSSION THREAD

**Discussion by zouyaoji** [zouyaoji/vue-cesium#89] [General]
## 👋 Welcome!

Welcome to vue-cesium Discussions!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shenhubeef** [comment]
关于热力图,动态对象和cesium+echarts的迁徙图要如何添加点击事件
